### DISCUSSION TITLE
How can i redirect the route to a new page in the plugin using NavigateTo

### DISCUSSION THREAD

**Discussion by inspce11** [nuxt/nuxt#22124] [Questions]
### Describe the feature

How can i redirect the route to a new page in the plugin using NavigateTo, 

### Additional information

- [ ] Would you be willing to help implement this feature?
- [ ] Could this feature be implemented as a module?

### Final checks

- [X] Read the [contribution guide](https://nuxt.com/docs/community/contribution).
- [X] Check existing [discussions](https://github.com/nuxt/nuxt/discussions) and [issues](https://github.com/nuxt/nuxt/issues).

**Comment by xjccc** [comment]
[middleware](https://nuxt.com/docs/api/utils/define-nuxt-route-middleware#redirection)
